src/pages/blog.js

```diff
@@ -15,7 +15,14 @@ function BlogPage(articulos) {
         <ul>
           {articulos.map((articulo) => (
             <li key={articulo.id}>
-              <Link to={`/blog/${articulo.slug}`}>{articulo.title}</Link>
+              <p>{articulo.title}</p>
+              <p>
+                {articulo.description_title}: {articulo.description}
+              </p>
+              <Link to={`/blog/${articulo.url_entry}`}>Leer más</Link>
```

Comment:
Aqui al redireccionar al articulo se debe quitar `/blog` del anchor del Link ya que rompe la direccion.
![image](https://user-images.githubusercontent.com/11642622/112890061-05b55580-909c-11eb-8027-6ced93138475.png)
![image](https://user-images.githubusercontent.com/11642622/112890137-1b2a7f80-909c-11eb-9f58-3a6698da73b2.png)


Debido a que  el **url_entry** ya agrega esa parte, esta linea deberia quedar asi, para que funcione igual.

```<Link to={`${articulo.url_entry}`}>Leer más</Link>```

![image](https://user-images.githubusercontent.com/11642622/112890270-48772d80-909c-11eb-8773-ae0e076e8e79.png)

Reply:
Si, se me había pasado por alto, ya lo voy a corregir.

Reply:
Genial @thegreatyamori 👍🏼, dale.
En el comentario anterior vi que hay cosas que faltan, pretendes agregarlas en este mismo PR?

Reply:
Quisiera añadir la entrada más leída, pero tengo que revisar la documentación de mdx para ver como se hace. 
Por otra parte quería preguntar si esta bien añadir la entrada de blog aquí o en un nuevo PR?

Reply:
Desafortunadamente, esto se puede lograr con analytics o cloud functions. Para obtener esa data del post más leido.
Investigando un poco encontré dos opciones:

### Añadir Analíticas
https://www.gatsbyjs.com/docs/how-to/adding-common-features/adding-analytics/
https://hippocampus-garden.com/trend/
https://decodenatura.com/how-to-add-a-trending-section-to-gatsby-blog/

### Usar Netlify Functions + Firebase
https://www.joshwcomeau.com/gatsby/using-netlify-functions-with-gatsby/
https://leerob.io/blog/real-time-post-views

Reply:
> Quisiera añadir la entrada más leída, pero tengo que revisar la documentación de mdx para ver como se hace.

Dale la ponemos como tarea sin resolver en el issue

> Por otra parte quería preguntar si esta bien añadir la entrada de blog aquí o en un nuevo PR?

Puedes hacerlo en este PR o en otro no hay problema.

Reply:
Lo voy a hacer en este PR entonces, mientras tanto voy resolviendo la convesación porque ya corregí el error.